core/pom.xml

```diff
@@ -247,7 +247,7 @@
             <groupId>org.commonjava.maven.ext</groupId>
             <artifactId>pom-manipulation-annotation</artifactId>
             <classifier>processor</classifier>
-            <version>3.9-SNAPSHOT</version>
+            <version>${project.version}</version>
```

Comment:
Weren't you saying recently not to use `${project.version}` and that maven-release-plugin would replace the version automatically anyway?

Reply:
Yes you're right, I shouldn't have changed that. Doh!